src/core/AnimatedTransform.js

```diff
@@ -54,19 +50,23 @@ class AnimatedTransform extends AnimatedNode {
     this._transform = transform;
   }
 
+  __setNewTransform(newTransform) {
+    this._transform = newTransform;
+  }
   __getProps() {
-    return this._transform.map(transform => {
-      const result = {};
+    const result = [];
```

Comment:
Previously Animated nodes have been passed to the transform map what was unnecessary